src/main/java/org/isf/lab/gui/LabEdit.java

```diff
@@ -583,12 +546,12 @@ private JButton getOkButton() {
 					return;
 				}
 				// exam date
-				GregorianCalendar gregDate = new GregorianCalendar();
-				ArrayList<String> labRow = new ArrayList<>();
-				RememberDates.setLastLabExamDate(gregDate);
+				LocalDateTime examDate = LocalDateTime.now();
+				List<String> labRow = new ArrayList<>();
+				RememberDates.setLastLabExamDate(examDate);
 
-				lab.setDate(gregDate);
-				lab.setExamDate(new GregorianCalendar());
+				lab.setDate(LocalDateTime.now());
+				lab.setExamDate(examDate.toLocalDate());
```

Comment:
here the hotfix was:
```
-				lab.setDate(new GregorianCalendar());
-				lab.setExamDate(gregDate);
+				lab.setDate(gregDate);
+				lab.setExamDate(new GregorianCalendar());
```
so it should be:
```
				lab.setDate(examDate.toLocalDate());
				lab.setExamDate(LocalDateTime.now());
```

Reply:
Actually it should be:
<pre>
				lab.setDate(examDate);
				lab.setExamDate(LocalDate.now());
</pre>
This is because `date` is `LocalDateTime` and `examDate` is just `LocalDate`.

Reply:
alright, thanks!